rio/src/RIO/Prelude/Exit.hs

```diff
@@ -0,0 +1,28 @@
+module RIO.Prelude.Exit
+  ( exitFailure
+  , exitSuccess
+  , exitWith
+  , System.Exit.ExitCode(..)
+  ) where
+
+import           Control.Monad.IO.Class
+import qualified System.Exit ( ExitCode (..)
+                             , exitFailure
+                             , exitSuccess
+                             , exitWith
+                             )
+
+--
+-- | Lifted version of "System.Exit.exitFailure", @since 0.1.9.0.
```

Comment:
This `@since` bits should be on their own lines to get them to render correctly.

Reply:
OK, updated.